deployment/cdk/opensearch-service-migration/lib/common-utilities.ts

```diff
@@ -466,6 +466,20 @@ export function parseClusterDefinition(json: any): ClusterYaml {
     return new ClusterYaml({endpoint, version, auth})
 }
 
+// eslint-disable-next-line @typescript-eslint/no-explicit-any
+export function parseSnapshotDefinition(json: any): SnapshotYaml {
+    const snapshotName = json.snapshotName
+    const s3Region = json.s3Region
+    const s3Uri = json.s3Uri
```

Comment:
double checking for our snapshot definition params: is there anywhere in the (pre-existing) code where we assume a specific snapshot repo name? Do we need to collect that as well?

Reply:
From my testing, snapshot repo name doesn't seem to matter when reading directly from S3 and Metadata and RFS seem to just need the details you highlighted as well as the version of the cluster.  It could be a problem if the user actually wanted to use the console snapshot commands on that external snapshot, but that seems a little odd since the snapshot should already be existing